### PR TITLE
ARROW-3719: [GLib] Support read/write table to/from Feather

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Comment by kou** [comment]
@shiro615 Can you review this?

File: c_glib/arrow-glib/reader.cpp [hunk 1]
```diff
@@ -801,6 +801,92 @@ garrow_feather_file_reader_get_columns(GArrowFeatherFileReader *reader,
   return g_list_reverse(columns);
 }
 
+/**
+ * garrow_feather_file_reader_read:
+ * @reader: A #GArrowFeatherFileReader.
+ * @error: (nullable): Return locatipcn for a #GError or %NULL.
```
Comment (shiro615) [inline c_glib/arrow-glib/reader.cpp]: typo? `locatipcn` => `location`
It seems to be using `locatipcn` in several places.

**Review by shiro615 — APPROVED** [review]
LGTM with nits

File: c_glib/arrow-glib/reader.cpp (hunk 1 above)
Comment (kou) [reply c_glib/arrow-glib/reader.cpp]: Good catch!

**Review by wesm — APPROVED** [review]
+1

**Pull request closed without merging**